config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/items", to: "items#index"
+
+  get "/supermarkets/:id", to: "supermarkets#show"
+
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customer_items#create"
```

Comment:
See my comment about your url in your form in the view. We could add to this route
```ruby
post "/customers/:id/items", to: "customer_items#create"
```